jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/strategy/AdaptiveExecutionStrategy.java

```diff
@@ -133,9 +133,9 @@ private enum SubStrategy
     private final LongAdder _epcMode = new LongAdder();
     private final LongAdder _epcProduce = new LongAdder();
     private final Producer _producer;
-    private final Executor _executor;
+    private final Executor _baseExecutor;
```

Comment:
"base" is confusing, not sure what it means?
I think like it was previously, we just want the original `Executor`, and the "virtual" `Executor`.

Reply:
It means the base executor from which the executor and tryExecutor are extracted. We must remember the original executor, as we cannot ask a virtual executor if it is a virtual executor.  I guess I could do it with a boolean...